script/results.html

```diff
@@ -63,6 +153,426 @@
+          document.getElementById("optional-tab-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-non-telco-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("myCheck3-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-non-telco-feedback").setAttribute("hidden", "hidden");
+
+          document.getElementById("myCheck2-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-faredge-feedback").setAttribute("hidden", "hidden");
+
+        showTelco()
+
+
+          document.getElementById("myCheck3-feedback").removeAttribute("hidden");
+          document.getElementById("optional-tab-telco-feedback").removeAttribute("hidden");
+
+    }
+    if (selectedValue=="nontelco"){
+          document.getElementById("myCheck3-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("myCheck1-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-telco-feedback").setAttribute("hidden", "hidden");
+
+          document.getElementById("myCheck2-feedback").removeAttribute("hidden");
+          document.getElementById("optional-tab-non-telco-feedback").removeAttribute("hidden");
+          showNonTelco()
+    }
+}
+
+function changeFunc() {
+    var selectBox = document.getElementById("selectBox");
+    var selectedValue = selectBox.options[selectBox.selectedIndex].value;
+
+    console.log("click on ",buttonElementId)
+
+    if (buttonElementId=="feedback-tab"){
+      enableoptionbuttonFeedback()
+
+      }
+      if (buttonElementId=="result-tab"){
+        console.log("click on result ")
+
+        enableoptionbutton()
+
+      }
+  }
+
+  function enableoptionbutton(){
+
+  buttonElementId = "result-tab"
+    var selectBox = document.getElementById("selectBox");
+    var selectedValue = selectBox.options[selectBox.selectedIndex].value;
+    console.log(selectedValue)
+    if (selectedValue=="all"){
+      showAll()
+    }
+    document.getElementById("optional-tab-feedback").setAttribute("hidden", "hidden");
+    document.getElementById("optional-tab-telco-feedback").setAttribute("hidden", "hidden");
+    document.getElementById("optional-tab-non-telco-feedback").setAttribute("hidden", "hidden");
+    document.getElementById("myCheck1-feedback").setAttribute("hidden", "hidden");
+    document.getElementById("myCheck2-feedback").setAttribute("hidden", "hidden");
+    document.getElementById("myCheck3-feedback").setAttribute("hidden", "hidden");
+    document.getElementById("send-all").setAttribute("hidden", "hidden");
+          document.getElementById("download-all").setAttribute("hidden", "hidden");
+          document.getElementById("download-NonTelco").setAttribute("hidden", "hidden");
```

Comment:
missing download-Telco? The Download button is not deleted in result page when telco scenario is selected

Reply:
yah did that now